src/python/tests/core/bot/fuzzers/libFuzzer/libfuzzer_launcher_integration_test.py

```diff
@@ -662,3 +664,119 @@ def test_merge_reductions(self, mock_get_timeout):
+        name='libFuzzer',
+        source='builtin',  # change to "test@example.com" if it acts up
+        max_testcases=4).put()
+
+    # Set up a FuzzerJob.
+    data_types.FuzzerJob(
+        fuzzer='libFuzzer',
+        job='libfuzzer_asan_test_fuzzer',
+        platform='FUCHSIA',
+        weight=1.0).put()
+
+    # Set up a FuzzTarget
+    data_types.FuzzTarget(
+        binary='libfuzzer_asan_test_fuzzer',
+        engine='libFuzzer',
+        project='test-project').put()
+
+    # Set up a FuzzTargetJob
+    data_types.FuzzTargetJob(
+        engine='libFuzzer',
+        fuzz_target_name='libFuzzer_libfuzzer_asan_test_fuzzer',
+        job='libfuzzer_asan_test_fuzzer',
+        weight=1.0).put()
+
+    # Set up a Job
+    data_types.Job(
+        environment_string=(
+            'CUSTOM_BINARY = True\n'
+            'FUCHSIA_RESOURCES_URL = gs://fuchsia-on-clusterfuzz-v2/*\n'
+            'QUEUE_OVERRIDE=FUCHSIA\n'
+            'OS_OVERRIDE=FUCHSIA'),
+        name='libfuzzer_asan_test_fuzzer',
+        platform='FUCHSIA',
+        templates=[u'libfuzzer', u'engine_asan']).put()
+
+    # Set up a JobTemplate
+    data_types.JobTemplate(
+        name='libfuzzer',
+        environment_string=('APP_NAME = launcher.py\n'
+                            'MAX_FUZZ_THREADS = 1\n'
+                            'MAX_TESTCASES = 4\n'
+                            'FUZZ_TEST_TIMEOUT = 4800\n'
+                            'TEST_TIMEOUT = 30\n'
+                            'WARMUP_TIMEOUT = 30\n'
+                            'BAD_BUILD_CHECK = False\n'
+                            'THREAD_ALIVE_CHECK_INTERVAL = 1\n'
+                            'REPORT_OOMS_AND_HANGS = True\n'
+                            'CORPUS_FUZZER_NAME_OVERRIDE = libFuzzer\n'
+                            'ENABLE_GESTURES = False\n'
+                            'THREAD_DELAY = 30.0')).put()
+
+    # Set up another JobTemplate
+    data_types.JobTemplate(
+        name='engine_asan',
+        environment_string=(
+            'LSAN = True\n'
+            'ADDITIONAL_ASAN_OPTIONS = quarantine_size_mb=64:strict_memcmp=1'
+            ':symbolize=0:fast_unwind_on_fatal=0'
+            ':allocator_release_to_os_interval_ms=500\n')).put()
+
+    environment.set_value('QUEUE_OVERRIDE', 'FUCHSIA')
+    environment.set_value('OS_OVERRIDE', 'FUCHSIA')
+    os.environ['FUCHSIA_RESOURCES_URL'] = 'gs://fuchsia-on-clusterfuzz-v2/*'
+    # set_bot_environment gives us access to RESOURCES_DIR
+    environment.set_bot_environment()
+    resources_dir = environment.get_value('RESOURCES_DIR')
+    if not resources_dir:
+      raise Exception('Could not find RESOURCES_DIR')
+    fuchsia_resources_dir = os.path.join(resources_dir, 'fuchsia')
+    pkey_path = os.path.join(fuchsia_resources_dir, '.ssh', 'pkey')
+    portnum = '56339'
+    environment.set_value('FUCHSIA_PKEY_PATH', pkey_path)
+    environment.set_value('FUCHSIA_PORTNUM', portnum)
+
+  def tearDown(self):
```

Comment:
This won't be necessary once we start patching the environment, delete this method.